src/main/scala/com/cloudera/sparkts/models/ARIMA.scala

```diff
@@ -806,3 +805,38 @@ class ARIMAModel(
     -2 * conditionalLogLikelihood + 2 * (p + q + interceptTerm)
   }
 }
+
+/**
+* Uses an eigenvalue decomposition to find the roots of a real-valued matrix, adapted for Apache Commons
+* Math 3. See: http://stackoverflow.com/questions/13805644/finding-roots-of-polynomial-in-java
+*/
+object PolynomialRootFinder {
```

Comment:
I think it would be best to have this functionality as a function inside of `ARIMA` instead of its own object.